tests/engagement_fields/snowplow_tests_unified_sessions.sql

```diff
@@ -0,0 +1,19 @@
+{#
```

Comment:
I think we could simplify these tests into one by adding additional parameter(s), once of which could be the model itself. You can read about it [here](https://docs.getdbt.com/best-practices/writing-custom-generic-tests) at the with additional arguments section.